### PR TITLE
Feature/ln user interaction

### PR DESCRIPTION
**Pull request by Alynn022**
PR Template:

Is this a fix or a feature?
- [ ] Bug fix
- [x] Feature
- [ ] Refactor
 
Describe the feature - fix - refactor: added favorite recipe button and feature so the user can see their favorite recipes.

How was this tested?
- [ ] open index.html
- [ ] localhost
- [x] terminal

Where should the reviewer start - test: script.js

Additional comments:
None.

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -34,90 +33,126 @@ const mainPageView = document.getElementById('mainPageView');
...
 // const randomRecipeImage = document.getElementById('randomRecipeImage');
 
-
 //CLASS INSTANSTIATION
 let cookbook = new RecipeBox(recipeData);
 let recipe = new Recipe(recipeData);
+let newUser;
+let currentRecipe; 
+
+// const getRandomIndex = (array) => {
```
Comment (StevieAmb) [inline src/scripts.js:54]: We need to remember to erase the code comments.

File: src/scripts.js [hunk 2]
```diff
@@ -34,90 +33,126 @@ const mainPageView = document.getElementById('mainPageView');
...
-
-const showAllRecipes = () => {
-  cookbook.recipeData.forEach(recipe => {
-    allRecipesView.insertAdjacentHTML('afterbegin', `
-    <article class="recipes-views" id= ${recipe.id}>
-    <h1>${recipe.name}</h1>
-    <img src="${recipe.image}">
-    `)
+  
+  allRecipesView.addEventListener('click', event => {
```
Comment (StevieAmb) [inline src/scripts.js:112]: This function is awesome!

**Review by StevieAmb — APPROVED** [review]

File: src/scripts.js (hunk 1 above)
Comment (Alynn022) [reply src/scripts.js:54]: Will do!

**Review by lschnell8 — APPROVED** [review]
Good work ya'll. I think we can combine the logic in the user class to add and remove recipes since it uses the same if statement. Also using an indexOf and assigning to a variable to pass it through the splice.

File: src/scripts.js (hunk 2 above)
Comment (Alynn022) [reply src/scripts.js:112]: Thanks to you!

**Pull request merged**